source/refresh-credentials/index.html.md.erb

```diff
@@ -0,0 +1,110 @@
+
+You must get the information about the user's existing credential from their user record. GOV.UK Wallet does not share any details about the user when they start your manual refresh journey.
+
+If GOV.UK Wallet adds the new credential successfully, the user will only see the new credential in their documents. GOV.UK Wallet will remove the old credential.
+
+## Understand GOV.UK Wallet's manual credential refresh flow
+
+This diagram shows the exchange between a government service (the 'credential issuer') and GOV.UK Wallet when the user refreshes a credential. This diagram, and the descriptions below it, are an explanation of each step in the process.
+
+<div aria-hidden="true">
+<script type="module">
+  import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@11/dist/mermaid.esm.min.mjs';
+</script>
+
+<html lang="en">
+  <body>
+    <pre class="mermaid">
+  sequenceDiagram
+
+  autonumber
+  actor user
+  participant ecri as Credential Issuer
+  participant app as App
+  participant ol as GOV.UK One Login
+
+  note over app,ol: On or near <br> credential expiry
+  user->>app: Opens GOV.UK One Login App <br> and goes to the credential
+  activate app
+    app->>ecri: Gets credential issuer metadata <br>`/.well-known/openid-credential-issuer`
+    activate ecri
+    ecri-->>app: Returns credential issuer metadata
+    deactivate ecri
+    app->>app: Finds `credential_refresh_web_journey_url` in metadata
+    app->>app: Validates `credential_refresh_web_journey_url`
+    app->>app: Shows refresh button to the user
+  deactivate app
+  user->>app: Clicks refresh button
+  app->>ecri: Accesses your service
+  activate ecri
+    ecri->>ol: Authenticates user
+    ol-->>ecri: Fetches user's walletSubjectID
+    ecri->>ecri: Generates credential offer
+    ecri-->>ecri: Renders credential offer<br>deep link
+    user->>ecri: Clicks deep link
+    ecri->>app: Launches app
+    activate app
+        app->>ecri: Redeems credential offer
+    deactivate app
+  deactivate ecri
+  </pre>
+    <script type="module">
+      import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@11/dist/mermaid.esm.min.mjs';
+    </script>
+  </body>
+</html>
+</div>
+
+1. Your user opens the app and finds the credential they need to refresh.
+
+1. GOV.UK Wallet sends a GET request to your `/.well-known/openid-credential-issuer` endpoint to fetch your credential issuer metadata.
+
+1. Your service returns its credential issuer metadata.
+
+1. The app finds the [`credential_refresh_web_journey_url`](../credential-issuer-functionality/metadata/#credential-information) value in the credential issuer metadata.
+
+1. The app validates the `credential_refresh_web_journey_url`.
+
+1. If the validation is successful, the app shows the user a refresh button.
+
+1. The user clicks the refresh button in the app.
```

Comment:
```suggestion
1. The user selects the 'Refresh' button in the app.
```

'selects' is preferable to 'clicks', and inverted commas should be used for UI elements in non-instructional contexts